File: CSE114/Homeworks/Homework1/CapitalLetterTester.java
//Edward Buckler
//111628438
//L05

import java.util.Scanner;

public class CapitalLetterTester {
    public static void main(String[] args) {
        System.out.println("Enter a string :");
        Scanner input = new Scanner(System.in);
        String inputtedString = input.nextLine();
        int UpperCase = 0;
        for (int i = 0; i < inputtedString.length(); i++){
            char currentChar = inputtedString.charAt(i);
            if(Character.isUpperCase(currentChar)){
                UpperCase++;
            }
        }
        System.out.println("The number of uppercase letters is "+UpperCase);
        input.close();
    }
}

File: CSE114/Homeworks/Homework1/CapitalLetterTester.kt
//Edward Buckler
//111628438
//L05

import java.util.Scanner

object CapitalLetterTester {
    @JvmStatic
    fun main(args: Array<String>) {
        println("Enter a string :")
        val input = Scanner(System.`in`)
        val inputtedString = input.nextLine()
        var UpperCase = 0
        for (i in 0 until inputtedString.length) {
            val currentChar = inputtedString[i]
            if (Character.isUpperCase(currentChar)) {
                UpperCase++
            }
        }
        println("The number of uppercase letters is $UpperCase")
        input.close()
    }
}
